README.md

```diff
@@ -62,7 +62,7 @@ This will give you access to
 
 Hitting these endpoints will `create`, `update` and `retrieve` instances of `UploadedFile` under the hood.
 
-**Please note that meta data can optionally be passed in at the initial post request, such as `file_key` and `filename`.**
+**Please note that meta data can optionally be passed in at the initial post request, such as `file_key` and `filename` and the `acl` type which will define the access control in S3.**
```

Comment:
Should we provide the list of options here too?

Reply:
Added

Reply:
Great thanks :)

Sorry to be a pain but we don't really need such a big title. Also, the formatting looks off.

Perhaps worth mentioning the default if no `acl` type is provided.

![Screenshot 2019-08-21 at 13 51 01](https://user-images.githubusercontent.com/12628901/63433323-dd269d80-c41a-11e9-992f-c1f9d46e47b8.png)

Reply:
Just think it's important to have clear instructions in the README as it is open source :)

Reply:
cool - updated